extension/modules/personas.js

```diff
@@ -1123,8 +1123,8 @@ var PersonaController = {
         // Update the Custom menu. Custom personas unavailable in per-window
         // personas mode.
         let customMenu = document.getElementById("personas-plus-custom-menu");
-        customMenu.hidden = this._prefs.get("perwindow") || !this._prefs.get("showCustomMenu");
-        if (!customMenu.hidden) {
+        customMenu.disabled = this._prefs.get("perwindow")
```

Comment:
OK, but then you need to handle this STR because Custom menu label is blank:

1. Create Custom Persona with name
2. Toggle `perwindow` pref to `true`
3. Check toolbar button menu and see the Custom label is empty